content/en/docs/reference/command-line-tools-reference/feature-gates/watch-list.md

```diff
@@ -9,5 +9,8 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.27"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
 ---
-Enable support for [streaming initial state of objects in watch requests](/docs/reference/using-api/api-concepts/#streaming-lists).
+Enable support for [streaming initial state of objects in watch requests](/content/en/docs/reference/using-api/api-concepts.md/#streaming-lists).
```

Comment:
The suggested link update seems unnecessary and incorrect. The existing link works as expected when rendered on the website. I'm not sure if this update is needed. 
Please check it out here: [https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates) _(check "WatchList" under the "List of feature gates" section)_.

The final redirection location is https://kubernetes.io/docs/reference/using-api/api-concepts/#streaming-lists.

Reply:
```suggestion
Enable support for [streaming initial state of objects in watch requests](/docs/reference/using-api/api-concepts/#streaming-lists).
```